catalog/app/containers/Package/index.js

```diff
@@ -199,7 +201,24 @@ Install.propTypes = {
   owner: PropTypes.string.isRequired,
 };
 
-const UpdateInfo = ({ author, date, version }) => (
+function readableBytes(bytes) {
+  const mb = bytes / 1000000;
+  const fracStr = (mb - Math.floor(mb)).toFixed(6);
```

Comment:
Move it inside the `if`? In fact, if `mb < 1`, you don't need to subtract anything.